stable/elasticsearch/templates/master-svc.yaml

```diff
@@ -1,6 +1,8 @@
 apiVersion: v1
 kind: Service
 metadata:
+  annotations:
+    service.alpha.kubernetes.io/tolerate-unready-endpoints: "true"
```

Comment:
Again I ask why are we effectively disabling the health checks to "fix" a service rather than fixing the broken health check?

Reply:
master service is only for discovery service and I think it's irrelevant to success of http health check because discovery service starts before http health check succeeded.

Reply:
I agree that we need to filter bad citizens.
Removed unready endpoints tolerations